explore-assistant-backend/terraform/variables.tfvars.sample

```diff
@@ -2,4 +2,8 @@ deployment_region = "asia-southeast1"
 project_id = "joon-sandbox"
 dataset_id_name = "explore_assistant"
 use_bigquery_backend=false
-use_cloud_function_backend=true
\ No newline at end of file
+use_cloud_function_backend=false
+use_cloud_run_backend=true
+cloud_run_service_name="explore-assistant-endpoint-ken"
+image_name="explore-assistant-endpoint-ken"
+image_tag="4f35ab1"
```

Comment:
we agreed to settle to latest tag, please remove

Reply:
removed